src/data/agile-quiz.ts

```diff
@@ -617,6 +617,58 @@ const agileQuiz = [
     Explanation:
       "Velocity is used to estimate the team's capacity and helps with sprint planning by forecasting how much work can be completed.",
     Link: "https://www.freecodecamp.org/news/complete-guide-to-agile-methodology/"
+  },
+  {
+    Question: "What's the difference between a burndown and a burnup chart?",
+    Answer:
+      "Burndown charts show how much work is left to be done versus the amount of time remaining, while burnup charts show the work that has been completed versus the total amount of work and time remaining.",
+    Distractor1:
+      "Burndown charts show the total amount of work completed versus the remaining time, while burnup charts show the work left to be done versus the time remaining.",
+    Distractor2:
+      "Burndown charts show the total amount of work completed versus the total amount of work, while burnup charts show the work that has been done versus the time remaining.",
+    Distractor3:
+      "Burndown charts show the work completed over a specific period, while burnup charts illustrate the work completed against the planned schedule and the actual time taken.",
```

Comment:
it is best to have shorter answers for the quiz layout because it look best not only in this app but also the freeCodeCamp discord app

```suggestion
    Answer:
      "Burndown charts show how much work is left to be done, while burnup charts show the work that has been completed",
    Distractor1:
      "Burndown charts show the total amount of work completed, while burnup charts show the work left to be done versus",
    Distractor2:
      "Burndown charts show the total amount of work completed, while burnup charts show the work that has been done",
    Distractor3:
      "Burndown charts show the work completed, while burnup charts show todo list items.",
```